std/shared/src/main/scala/cats/effect/std/UUIDGen.scala

```diff
@@ -59,4 +137,11 @@ object UUIDGen extends UUIDGenCompanionPlatform {
         new UUID(msb, lsb)
       }
     }
+
+  private[std] abstract class TranslatedUUIDGen[F[_], G[_]](self: UUIDGen[F])(f: F ~> G)
```

Comment:
This can probably be a `final` class right?

```suggestion
  private[std] final class TranslatedUUIDGen[F[_], G[_]](self: UUIDGen[F])(f: F ~> G)
```

Reply:
Yeah, I agree with you. To be honest, not sure why the similar one `Random.TranslatedRandom` is abstract.

Reply:
> To be honest, not sure why the similar one `Random.TranslatedRandom` is abstract.

I'm not entirely sure either, but probably related to this.

https://github.com/typelevel/cats-effect/blob/0cb794be902b39d0e43ae0ad3a44e10c94edabff/std/shared/src/main/scala/cats/effect/std/SecureRandom.scala#L43-L44